apps/strapi-cms/src/api/quickstart-guide-item/content-types/quickstart-guide-item/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "kind": "collectionType",
+  "collectionName": "quickstart_guide_items",
+  "info": {
+    "singularName": "quickstart-guide-item",
+    "pluralName": "quickstart-guide-items",
+    "displayName": "QuickstartGuideItem"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
+  },
+  "attributes": {
+    "quickstart_guide": {
+      "type": "relation",
+      "relation": "manyToOne",
+      "target": "api::quickstart-guide.quickstart-guide",
+      "inversedBy": "quickstart_guide_items"
+    },
+    "title": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string"
+    },
+    "index": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string"
+    },
+    "content": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "blocks"
+    }
```

Comment:
Some attribute are missing and i think `content` should be more complex in NextJS `QuickStartGuide` type manage multiple steps `apps/nextjs-website/src/lib/types/step.ts` and in each step can show different types of bloc `apps/nextjs-website/src/lib/types/part.ts`